### PR TITLE
Add some documentation about annotations.

### PR DESCRIPTION
**Pull request by manthey**
Added an endpoint to return the official schema.

### CODE REVIEW

File: docs/annotations.md [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+
+```
+{
+    "type": "point",                  # Exact string for the specific shape.  Required
+    "id": "0123456789abcdef01234567", # String, 24 lowercase hexadecimal digits.  Optional.
+    "label": {                        # Object.  Optional
+      "value": "This is a label",     # String.  Optional
+      "visability": "hidden",         # String.  One of "always", "hidden", "onhover".  Optional
+      "fontSize": 3.4,                # Number.  Optional
+      "color": "#0000FF"              # String.  See note about colors.  Optional
```
Comment (jbeezley) [inline docs/annotations.md]: Does this mean the color of the font, the label background, or something else?

File: docs/annotations.md [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+    "type": "rectanglegrid",           # Exact string.  Required
+    <id, label, lineColor, lineWidth>  # Optional general shape properties
+    "center": [10.3, -40.0, 0],        # Coordinate.  Required
+    "width": 5.3,                      # Number >= 0.  Required
+    "height": 17.3,                    # Number >= 0.  Required
+    "rotation": 0,                     # Number.  Counterclockwise radians around normal.  Required
+    "normal": [0, 0, 1.0],             # Three numbers specifying normal.  Default is positive Z.  Optional
+    "widthSubdivisions": 3,            # Integer > 0.  Required
+    "heightSubdivisions": 4,           # Integer > 0.  Required
+    "fillColor": "rgba(0, 255, 0, 1)"  # String.  See note about colors.  Optional
```
Comment (jbeezley) [inline docs/annotations.md]: Are grids supposed to have fill colors?

File: docs/annotations.md (hunk 1 above)
Comment (manthey) [inline docs/annotations.md]: Font color (though we might have to as @brianhelba to be certain).  I don't think that we currently have a concept of a label bound box and background (or even relative positioning).

File: docs/annotations.md (hunk 2 above)
Comment (manthey) [inline docs/annotations.md]: I doubt @law12019 is using a fill color at all in his use of rectanglegrids.  I could see wanting the whole region to have a color with a partial opacity.

Comment (jbeezley) [inline docs/annotations.md]: Yes, I suppose if the grid can be a subregion of the full image, than it could be useful.

**Comment by cdeepakroy** [comment]
LGTM!

**Pull request merged**